examples/sensortag.py

```diff
@@ -137,7 +137,7 @@ async def run(address, debug=False):
         battery_level = await client.read_gatt_char(BATTERY_LEVEL_UUID)
         print("Battery Level: {0}%".format(int(battery_level[0])))
 
-        def keypress_handler(sender, data):
+        async def keypress_handler(sender, data):
```

Comment:
It would be nice if we had an example that actually awaited something. Can't think of anything of-hand though.

Reply:
I was thinking I maybe needed to write an example that unloaded data to a async queue. Was planning to wait for the docs work, but it might just fit here better.

Reply:
Added said example.